src/ServicePulse.Host/app/js/views/failed_groups/controller.js

```diff
@@ -123,8 +123,8 @@
             return group.workflow_state.status !== 'none' && (group.workflow_state.status !== 'completed' || group.need_user_acknowledgement === true) && !vm.isBeingArchived(group.workflow_state.status);
         };
 
-        vm.isBeingArchived = function (status) {
-            return status === "archivestarted" || status === "archiveprogressing" || status === "archivefinalizing" || status === "archivecompleted";
+        vm.isBeingArchived = function (group) {
```

Comment:
This fix require some more work. 
 * There are at least 2-3 places where we still call isBeingArchived passing status not a  group (easy fix)
 * `operation_status` that you check contains status that is not to lower, so either tha names need to be adjusted or tolower used (easy fix)
 * `operation_status` is not being updated by our event handlers. So only when the list is refreshed the state will be actual. So we should update operation_status field whenever an event is handled.